plugins/chat/assets/javascripts/discourse/components/chat-composer-uploads.js

```diff
@@ -26,12 +26,14 @@ export default Component.extend(UppyUploadMixin, {
 
   didReceiveAttrs() {
     this._super(...arguments);
+    if (this._uppyInstance?.getState?.()?.totalProgress > 0) {
```

Comment:
This is a micro optim, but it avoids doing all kind of unecessary work like sending an app event